easybuild/easyconfigs/t/Trilinos/Trilinos-12.12.1-foss-2019b-Python-3.7.4.eb

```diff
@@ -10,11 +10,8 @@ description = """The Trilinos Project is an effort to develop algorithms and ena
 toolchain = {'name': 'foss', 'version': '2019b'}
 toolchainopts = {'usempi': True, 'pic': True, 'strict': True}
 
-source_urls = [
-    'http://trilinos.csbsju.edu/download/files/',
-    'https://trilinos.org/oldsite/download/files/',
-]
-sources = ['%(namelower)s-%(version)s-Source.tar.gz']
+source_urls = ['https://launchpad.net/ubuntu/+archive/primary/+sourcefiles/trilinos/12.12.1-5/']
```

Comment:
There is a tagged release for v12.12.1 in their github repository at https://github.com/trilinos/Trilinos/archive/refs/tags/trilinos-release-12-12-1.tar.gz . Should we used this one instead of the tarball of a third-party?

Reply:
Discussion in #14066